.travis.yml

```diff
@@ -1,3 +1,15 @@
 language: node_js
 node_js:
-  - "6.9"
+  - stable
```

Comment:
I _think_ that you don't need this top level `node_js: - stable` if you have them also in the matrix below. Aside from that, I think you should set the node version (probably to 10) rather than put it at stable so that we know it won't break if they change something in node.

Reply:
I changed it to 12.8.0 because that's the version I'm using and testing on.